internal/router/router.go

```diff
@@ -260,13 +260,17 @@ func New(
 		Context:  ctxt,
 	}
 	mux := NewServeMux()
-	mux.Handle("/meta/", http.StripPrefix("/meta", HandleErrors(r.serveBulkMeta)))
+	mux.Handle("/meta/", http.StripPrefix("/meta", http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
+		r.Monitor.SetEndpoint(fmt.Sprintf("/meta%s", req.URL.Path))
```

Comment:
Sprintf is just a distraction here:

    r.Monitor.SetEndpoint("/meta" + req.URL.Path)